app.py

```diff
@@ -12,16 +50,12 @@ def respond(
     hf_token: gr.OAuthToken,
 ):
     """
-    For more information on `huggingface_hub` Inference API support, please check the docs: https://huggingface.co/docs/huggingface_hub/v0.22.2/en/guides/inference
+    Original function for LLM interaction via the Hugging Face Inference API.
```

Comment:
Is there a reason why you removed the link to the HF Inference API documentation which was present in the template? Otherwise I would leave it